stetho/src/main/java/com/facebook/stetho/common/android/FragmentApi.java

```diff
@@ -0,0 +1,459 @@
+    @Override
+    public List<?> getAddedFragments(Object fragmentManager) {
+      return null;
+    }
+  }
+
+  private static final class ReflectionFragmentAccessor implements FragmentAccessor {
+    // We access the field instead of calling the getChildFragmentManager() method
+    // because the latter will instantiate one if it doesn't exist
+    private final Field mFieldMChildFragmentManager;
+    private final Method mMethodGetFragmentManager;
+    private final Method mMethodGetId;
+    private final Method mMethodGetTag;
+    private final Method mMethodGetView;
+
+    public ReflectionFragmentAccessor(Class<?> fragmentClass) {
+      Util.throwIfNull(fragmentClass);
+
+      Field fieldMChildFragmentManager;
+      try {
+        fieldMChildFragmentManager = fragmentClass.getDeclaredField("mChildFragmentManager");
+        fieldMChildFragmentManager.setAccessible(true);
+      } catch (Exception e) {
+        LogUtil.w(
+            e,
+            "Could not retrieve mChildFragmentManager field from %s",
+            fragmentClass);
+
+        fieldMChildFragmentManager = null;
+      }
+
+      Method methodGetFragmentManager;
+      try {
+        methodGetFragmentManager = fragmentClass.getMethod("getFragmentManager");
+      } catch (Exception e) {
+        LogUtil.w(
+            e,
+            "Could not retrieve getFragmentManager() method from %s",
+            fragmentClass);
+
+        methodGetFragmentManager = null;
+      }
+
+      Method methodGetId;
+      try {
+        methodGetId = fragmentClass.getMethod("getId");
+      } catch (Exception e) {
+        LogUtil.w(
+            e,
+            "Could not retrieve getId() method from %s",
+            fragmentClass);
+
+        methodGetId = null;
+      }
+
+      Method methodGetTag;
+      try {
+        methodGetTag = fragmentClass.getMethod("getTag");
+      } catch (Exception e) {
+        LogUtil.w(
+            e,
+            "Could not retrieve getTag() method from %s",
+            fragmentClass);
+
+        methodGetTag = null;
+      }
+
+      Method methodGetView;
+      try {
+        methodGetView = fragmentClass.getMethod("getView");
+      } catch (Exception e) {
+        LogUtil.w(
+            e,
+            "Could not retrieve getView() method from %s",
+            fragmentClass);
+
+        methodGetView = null;
+      }
+
+      mFieldMChildFragmentManager = fieldMChildFragmentManager;
+      mMethodGetFragmentManager = methodGetFragmentManager;
+      mMethodGetId = methodGetId;
+      mMethodGetTag = methodGetTag;
+      mMethodGetView = methodGetView;
+    }
+
+    @Override
+    public Object getFragmentManager(Object fragment) {
+      if (mMethodGetFragmentManager == null) {
+        return null;
+      }
+
+      try {
+        return mMethodGetFragmentManager.invoke(fragment);
+      } catch (Exception e) {
+        LogUtil.w(e, "Could not call Fragment.getFragmentManager()");
```

Comment:
The advice applies here as well on account of the fact that you've already proven the method exists.  It must be invokable else something very unusual has occurred that we want to know about.